style.css

```diff
@@ -0,0 +1,548 @@
+    transition: opacity 0.3s;
+}
+.footer-navigation__link:hover {
+    opacity: 1;
+}
+.copyright {
+    text-align: center;
+    font-size: 12px;
+    opacity: 0.7;
+    margin-top: 40px;
+    width: 100%;
+}
+.footer-navigation__icons {
+    margin-right: 15px;
+}
+.footer-navigation__icons img {
+    width: 24px;
+    height: 24px;
+    opacity: 0.7;
+    transition: opacity 0.3s;
+}
+.footer-navigation__icons img:hover {
+    opacity: 1;
+}
+h2.footer {
+  color:blue 
+}
+/*footer end*/
+/*binabox start*/
+h1.binabox {
+  font-family: Bakbak One;
+  font-weight: 400px;
+  font-size: 52px;
+  line-height: 62px;
+  letter-spacing: -0.5px;
+}
+h1 span {
+  color: #0265dc;
+}
+.binabox__title {
+  text-align: center;
+  font-family: Poppins;
+  font-weight: 500px;
+  font-size: 20px;
+  line-height: 28px;
+  letter-spacing: 0px;
+}
+.binabox__details {
+  font-family: Bakbak One;
+  font-weight: 400px;
+  font-size: 52px;
+  line-height: 62px;
+  letter-spacing: -0.5px;
+}
+.binabox__details small {
+  font-family: Poppins;
+  font-weight: 400px;
+  font-size: 16px;
+  line-height: 24px;
+  letter-spacing: 0px;
+}
+.binabox__button {
+  width: 190px;
+  height: 48px;
+  padding-top: 10px;
+  padding-right: Spacing24;
+  padding-bottom: 10px;
+  padding-left: Spacing24;
+  background: #0265dc;
+}
+/*binabox end*/
+/* catalog */
+catalog{ display: grid;
+  flex-wrap: wrap;
+  gap: 20px;
+  justify-content: center;
+  padding: 20px;}
+h2.catalog {
+  width: 115px;
+  height: 36px;
+  gap: Spacing8;
+  padding-top: 2px;
+  padding-bottom: 2px;
+}
+.catalog__button .catalog-button__see {
+  justify-content: end;
+}
+.catalog__button--see {
+  width: 108px;
+  height: 48px;
+  gap: Spacing8;
+  padding-top: 10px;
+  padding-right: Spacing24;
+  padding-bottom: 10px;
+  padding-left: Spacing24;
+  border-width: 2px;
+  cursor: pointer;
+  align-items: right;
+  color: rgba(245, 248, 251, 1);
+  background-color: transparent;
+}
+.catalog__button {
+  background-color: transparent;
+  color: white;
+  border: 1px solid white;
+  padding: 5px 10px;
+  cursor: pointer;
+  margin-left: 5px;
+}
+.product-card {
+  background-color: rgba(29, 28, 40, 1);
+  width: 300.79998779296875;
+  height: 480px;
+  gap: Spacing/Spacing24;
+  padding: Spacing/Spacing24;
+  display: inline-block;
+  text-align: center;
+  position: relative;
+}
+.product-card__image {
+  width: 252.79998779296875px;
+  height: 324px;
+}
+.product-card__badge {
+  position: absolute;
+  background-color: rgba(245, 248, 251, 1);
+  color: rgba(2, 101, 220, 1);
+  width: 51px;
+  height: 36px;
+  gap: 10px;
+  padding-top: 8px;
+  padding-right: 10px;
+  padding-bottom: 8px;
+  padding-left: 10px;
+}
+.product-card__badge--soldout {
+  background-color: rgba(234, 56, 41, 1);
+  width: 78px;
+  height: 36px;
+  gap: 10px;
+  padding-top: 8px;
+  padding-right: 10px;
+  padding-bottom: 8px;
+  padding-left: 10px;
+}
+.product-card__price {
+  font-family: Bakbak One;
```

Comment:
fallback font should be here and in another places where "font-family" is used

Reply:
"fallback" is not the name of font, it's an approach - https://css-tricks.com/css-basics-fallback-font-stacks-robust-web-typography/